<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientNewDialog</class>
 <widget class="QDialog" name="PatientNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PatientNewDialog</string>
  </property>
  <widget class="QWidget" name="widgetNewAna" native="true">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>260</y>
     <width>816</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>158</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>158</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生日：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sex">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>88</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>26</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>88</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>158</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>175</width>
      <height>175</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>88</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>28</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>错构瘤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>癫痫</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>帕金森</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>肿瘤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>28</y>
      <width>95</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>分析名称：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>210</y>
      <width>746</width>
      <height>168</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnAddIcon">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>66</y>
       <width>37</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加备注信息</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorRemark">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>746</width>
       <height>167</height>
      </rect>
     </property>
    </widget>
    <zorder>doctorRemark</zorder>
    <zorder>btnAddIcon</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>28</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>疾病种类：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>88</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_birthday">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>158</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>864</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #525F7B;border-radius: 8px 8px 8px 8px;</string>
   </property>
   <property name="text">
    <string>    新建分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>864</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #363D4A;border-radius: 8px 8px 8px 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>670</y>
     <width>100</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>670</y>
     <width>100</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>200</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>widgetNewAna</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_title</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
